src/components/staking/components/StakingAction.js

```diff
@@ -169,7 +183,13 @@ export default function StakingAction({
                     />
                 }
                 <div className='desc'><Translate id={`staking.${action}Success.descTwo`}/></div>
-                <FormButton linkTo='/staking' className='gray-blue'><Translate id={`staking.${action}Success.button`} /></FormButton>
+                <FormButton 
```

Comment:
looks like in many cases we are tracking button clicks
I think we should just customise respective components like `FormButton`  so that you don't have to repeat boilerplate every time.


so instead of 
```jsx
<FormButton 
                     linkTo='/staking' 
                     className='gray-blue'
                     onClick={() => Mixpanel.track("STAKE/UNSTAKE Return to dashboard")}
                 >
```

you can have:

```jsx
<FormButton linkTo='/staking' className='gray-blue' trackingId='STAKE/UNSTAKE Return to dashboard'><Translate id={`staking.${action}Success.button`} /></FormButton>
```

Reply:
will change this too